docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -244,4 +244,1052 @@ You have a general idea of what nf-core offers and you know how to run an nf-cor
+               ╷                                                                       ╷
+ 📥 Inputs     │Description                                                            │     Pattern
+╺━━━━━━━━━━━━━━┿━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┿━━━━━━━━━━━━╸
+ input[0]      │                                                                       │
+╶──────────────┼───────────────────────────────────────────────────────────────────────┼────────────╴
+  meta  (map)  │Groovy Map containing sample information e.g. [ id:'test',             │
+               │single_end:false ]                                                     │
+╶──────────────┼───────────────────────────────────────────────────────────────────────┼────────────╴
+  reads  (file)│List of input FastQ files                                              │*.{fastq.gz}
+               ╵                                                                       ╵
+                      ╷                                                                ╷
+ 📥 Outputs           │Description                                                     │     Pattern
+╺━━━━━━━━━━━━━━━━━━━━━┿━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┿━━━━━━━━━━━━╸
+ reads                │                                                                │
+╶─────────────────────┼────────────────────────────────────────────────────────────────┼────────────╴
+  meta  (map)         │Groovy Map containing sample information e.g. [ id:'test',      │
+                      │single_end:false ]                                              │
+╶─────────────────────┼────────────────────────────────────────────────────────────────┼────────────╴
+  *.fastq.gz  (file)  │Filtered FastQ files                                            │*.{fastq.gz}
+╶─────────────────────┼────────────────────────────────────────────────────────────────┼────────────╴
+ versions             │                                                                │
+╶─────────────────────┼────────────────────────────────────────────────────────────────┼────────────╴
+  versions.yml  (file)│File containing software versions                               │versions.yml
+                      ╵                                                                ╵
+
+ Use the following statement to include this module:
+
+ include { SEQTK_TRIM } from '../modules/nf-core/seqtk/trim/main'
+```
+
+Using this module information you can work out what inputs are required for the `SEQTK_TRIM` process:
+
+1.  `tuple val(meta), path(reads)`
+
+    -   A tuple with a meta _map_ and a list of FASTQ _files_
+    -   The channel `ch_samplesheet` used by the `FASTQC` process can be used as the reads input.
+
+As only one input channel required, and it already exists, it can be added to your `firstpipeline.nf` file without any additional channel creation or modifications.
```

Comment:
```suggestion
Only one input channel is required, and it already exists, so it can be added to your `firstpipeline.nf` file without any additional channel creation or modifications.
```